packages/editor/src/components/post-schedule/index.js

```diff
@@ -33,6 +33,9 @@ export default compose( [
 	withSelect( ( select ) => {
 		return {
 			date: select( 'core/editor' ).getEditedPostAttribute( 'date' ),
+			modified: select( 'core/editor' ).getEditedPostAttribute(
+				'modified'
```

Comment:
what is this "modified" attribute? and why it's being used here? Also, it seems that this change is unrelated with the root issue of the PR (unschedule link)?

Reply:
`modified` is the timestamp the post was last modified. `modified === date` is one of the ways we determine that the post is considered to have a floating date. 

If you pass a `null` value for `currentDate` to the `DateTimePicker` then it uses the current date to pre-populate the input fields. Then `onBlur` it will save the current value of the input as `date`, effectively eliminating the floating date we just set. It immediately closes and uses the current time no matter what you click inside it. 

The REST API expects `null` in order to make a date floating, so that's why we're using that for the date.